client/state/selectors/test/get-jetpack-product-install-progress.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import getJetpackProductInstallProgress from 'state/selectors/get-jetpack-product-install-progress';
+
+describe( '#getJetpackProductInstallProgress', () => {
+	const siteId = 12345678;
+
+	test( "should return `null` when we don't have the product installation progress for this site", () => {
+		expect( getJetpackProductInstallProgress( {}, siteId ) ).toBeNull();
+	} );
+
+	test( 'should return the product installation progress when we have it for this site', () => {
+		const state = {
+			jetpackProductInstall: {
+				[ siteId ]: {
+					akismet_status: 'key_not_set',
+					progress: 60,
+					vaultpress_status: 'key_not_set',
+				},
+			},
+		};
+		expect( getJetpackProductInstallProgress( state, siteId ) ).toEqual( 60 );
+	} );
+} );
```

Comment:
The selector implementations are so trivial we're testing the implementation of `get` and JS internals operating on a known object. I'd be happy to remove these.

Reply:
Yeah, that's a valid point. But still I prefer keeping them, - even if they don't have such a high value as usual selector tests, they demonstrate the data that's expected for this selector to return, which can be useful in lack of context.